### PR TITLE
Add default economic strategy params

### PR DESCRIPTION
**Pull request by Bagaric**

### CODE REVIEW

File: src/Config/Config.ts [hunk 1]
```diff
@@ -36,6 +37,13 @@ export default class Config implements IConfigParams {
       throw new Error('Please set the providerUrl in the config object.');
     }
 
+    this.economicStrategy = params.economicStrategy || {
```
Comment (adibas03) [inline src/Config/Config.ts]: will the values be visible to the user from here?
We need the user to see the default values if they open the `Economic strategy` section when trying to activate claiming

Comment (Bagaric) [reply src/Config/Config.ts]: This is up to the clients (cli/dapp) to decide how they want to show it to the user. They can always access `config.economicStrategy.maxDeposit` and check, etc.

Comment (Bagaric) [reply src/Config/Config.ts]: We could also expose it so they can show it to the user even before setting the Config object.

Comment (Bagaric) [reply src/Config/Config.ts]: I would like to avoid that though, adds too much complexity.

**Pull request merged**